### PR TITLE
Cover pkg/recovery

### PR DESCRIPTION
**Pull request by MDr164**
Signed-off-by: Marvin Drees <marvin.drees@9elements.com>

### CODE REVIEW

File: pkg/recovery/securerecoverer_test.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		},
+		{
+			name: "recover reboot failed",
+			rec: SecureRecoverer{
+				Reboot: true,
+			},
+		},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			if tt.name == "recover reboot" {
```
Comment (depate) [inline pkg/recovery/securerecoverer_test.go:55]: Seriously?! ;-)

**Review by depate — CHANGES_REQUESTED** [review]
if conditions...

File: pkg/recovery/securerecoverer_test.go (hunk 1 above)
Comment (jensdrenhaus) [reply pkg/recovery/securerecoverer_test.go]: +10000

**Review by depate — DISMISSED** [review]
Smells good to me

**Review by jensdrenhaus — DISMISSED** [review]

**Review by ChriMarMe — APPROVED** [review]

**Review by ChriMarMe — CHANGES_REQUESTED** [review]
Remove changes to go.yml

And rebase

**Review by llogen — COMMENTED** [review]
Nothing to add.

**Review by ChriMarMe — DISMISSED** [review]

**Pull request closed without merging**